components/layout_2020/display_list/stacking_context.rs

```diff
@@ -414,3 +490,52 @@ impl AnonymousFragment {
         }
     }
 }
+
+pub trait ToLayout {
```

Comment:
Could this reuse the `geom::ToWebrender` trait instead? (And perhaps move it to some other module since it’s not geometry-specific. Perhaps `display_list/mod.rs` since that’s the module that interacts most with WebRender?)

Reply:
Great idea. I've moved this to a "conversions" module that is similar to the one in layout_2013.